sycl/unittests/Extensions/FreeFunctionCommands/FreeFunctionCommandsEvents.cpp

```diff
@@ -21,6 +21,22 @@ class TestFunctor {
   void operator()(sycl::item<1>) const {}
   void operator()(sycl::nd_item<1> Item) const {}
 };
+
+class TestMoveFunctor {
+public:
+  static int MoveCtorCalls;
+
+  TestMoveFunctor() = default;
+  TestMoveFunctor(const TestMoveFunctor &) = default;
+  TestMoveFunctor(TestMoveFunctor &&) { ++MoveCtorCalls; }
+  void operator()() const {}
+  void operator()(sycl::item<1>) const {}
+  void operator()(sycl::nd_item<1> Item) const {}
+  void operator()(sycl::nd_item<3> Item) const {}
+};
+
+int TestMoveFunctor::MoveCtorCalls;
```

Comment:
```suggestion
int TestMoveFunctor::MoveCtorCalls = 0;
```

IIUC, still need explicit zero here.